server/lib/utils.js

```diff
@@ -5,11 +5,14 @@ let path = require("path");
 
 class Utils {
   static getFileList(root, pattern) {
-    return glob.sync(pattern, {
-      cwd: root,
-      matchBase: true,
-      nodir: true
-    }).map(file => path.join(root, file));
+    try {
+      return glob.sync(pattern, {
+        cwd: root,
+        matchBase: true,
+        nodir: true
+      }).map(file => path.join(root, file));
+    }
+    catch (err) { }
```

Comment:
Never, ever, ever.  Not ever.  Never. Not even once.

This pattern will cause you endless amounts of grief, when you have some bug where it silently fails and you don't know why.  Always do something with the error.